[2022-12-26T04:05:44.592+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-26T04:05:41.853800+00:00 [queued]>
[2022-12-26T04:05:44.600+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-26T04:05:41.853800+00:00 [queued]>
[2022-12-26T04:05:44.600+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T04:05:44.601+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-26T04:05:44.601+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T04:05:44.664+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table> on 2022-12-26 04:05:41.853800+00:00
[2022-12-26T04:05:44.670+0000] {standard_task_runner.py:55} INFO - Started process 190 to run task
[2022-12-26T04:05:44.672+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'new_table_and_data_import_dag_v1', 'create_table', 'manual__2022-12-26T04:05:41.853800+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/new_table_and_data_import_dag.py', '--cfg-path', '/tmp/tmp8w5ktung']
[2022-12-26T04:05:44.674+0000] {standard_task_runner.py:83} INFO - Job 71: Subtask create_table
[2022-12-26T04:05:44.759+0000] {task_command.py:389} INFO - Running <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-26T04:05:41.853800+00:00 [running]> on host 3a7da4d7a43a
[2022-12-26T04:05:45.033+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=new_table_and_data_import_dag_v1
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-26T04:05:41.853800+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-26T04:05:41.853800+00:00
[2022-12-26T04:05:45.034+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        
[2022-12-26T04:05:45.041+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-26T04:05:45.274+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-26T04:05:45.292+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        , parameters: None
[2022-12-26T04:05:46.886+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=new_table_and_data_import_dag_v1, task_id=create_table, execution_date=20221226T040541, start_date=20221226T040544, end_date=20221226T040546
[2022-12-26T04:05:47.049+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-26T04:05:47.093+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
